.github/workflows/docs-check.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - name: Check helm-docs
         run: make check-helm-docs
       - name: Check docs
-        run: make check-docs
+        run: make check.docs
```

Comment:
This command needs to be changed in the documentation as well - https://github.com/rook/rook/blob/master/Documentation/Contributing/documentation.md#making-docs

Reply:
done